suncity/suncity_resources.ini

```diff
@@ -35,35 +61,45 @@ GENOME_SOURCE=1000G
 GENOME_FASTA_DOWNLOAD_LINK=ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/technical/reference/phase2_reference_assembly_sequence/hs37d5.fa.gz
 GENOME_FASTA_MD5_DOWNLOAD_LINK=NA
 GENOME_ASSEMBLY_NAME=GRCh37
-GENOME_SUBVERSION_NAME=hs37d5_suncity
+GENOME_SUBVERSION_NAME=hs37d5
 
 ###########################################
 ##  BWA Index Variables
 ###########################################
 
 REFERENCE_DNA_GENOME_FASTA=/home/tgenref/homo_sapiens/grch37_hg19/hs37d5_suncity/genome_reference/hs37d5.fa
 REFERENCE_DNA_GENOME_BASENAME=/home/tgenref/homo_sapiens/grch37_hg19/hs37d5_suncity/genome_reference/hs37d5
+REFERENCE_DNA_GENOME_NAME=hs37d5.fa
+REFERENCE_RNA_GENOME_NAME=hs37d5.fa
```

Comment:
same RNA genome redundancy @bryce-turner

Reply:
We need both of these definitions for compatibility with the shared resources scripts. Since we required separate definitions for Phoenix we also need separate definitions in the other ini files.